control-plane/p4RuntimeSerializer.cpp

```diff
@@ -754,6 +761,9 @@ class P4RuntimeSymbolTable {
             case P4RuntimeSymbolType::COUNTER: return PI_COUNTER_ID;
             case P4RuntimeSymbolType::METER: return PI_METER_ID;
             case P4RuntimeSymbolType::TABLE: return PI_TABLE_ID;
+            // XXX(antonin): no support from PI library for now
+            // the choice for the resource type id has little importance here
+            case P4RuntimeSymbolType::CONTROLLER_HEADER: return 0xab;
```

Comment:
I ran into this same situation in my own recent patch. I suppose we have a number of these resource types to add at this point.